src/components/Ticketbox/index.jsx

```diff
@@ -68,13 +56,18 @@ const TicketboxButtons = ({ switchRenderedType }) => {
       }
     </div>
   )
-}
+};
+
+TicketboxButtons.propTypes = {
+  switchRenderedType: PropTypes.func.isRequired,
+};
 
 // Disable TicketBox on HepPage for non-auth users
-export default withRouter(({ location }) =>
-  location.pathname !== "/help" &&
-  <PopupMenu className="ticketbox-container">
-    <div className="ticketbox-btn--main">?</div>
-    <TicketboxPopup />
-  </PopupMenu>
-);
+export default () => localStorage.token
+  ? (
+      <PopupMenu className="ticketbox-container">
+        <div className="ticketbox-btn--main">?</div>
+        <TicketboxPopup />
+      </PopupMenu>
+    )
+  : null; // only render if the user is logged in
```

Comment:
Did we abandon the idea to show the tickebox help search for non logged in users?

Reply:
ya were putting it on hold for now until a future sprint

Reply:
lets comment out that link entirely until we have it ready i nthe future sprint